framework/rbac/BaseManager.php

```diff
@@ -190,6 +190,20 @@ public function getRoles()
     }
 
     /**
+     * Returns defaultRoles as array of Role objects
+     * @since 2.0.12
+     * @return Role[] default roles. The array is indexed by the role names
+     */
+    public function getDefaultRoles()
```

Comment:
@developeruz Yes, you should create new PR. Suggest your variation of naming and we can discuss from there.